docroot/modules/custom/mass_content/mass_content.deploy.php

```diff
@@ -317,3 +317,10 @@ function mass_content_deploy_header_media_images_followup(&$sandbox) {
     return t('All Header Media image caption fields migrated.');
   }
 }
+
+/**
+ * Migrate unpublished Header Media image wrapping fields.
+ */
+function mass_content_deploy_header_media_images_unpublished(&$sandbox) {
+  mass_content_deploy_header_media_images_followup($sandbox, 0);
```

Comment:
I modified the previous deploy hook to allow a status argument. The only difference between both field migrations is status, and in this case we want to only modify unpublished content.